squad/core/management/commands/migrate_attachments.py

```diff
@@ -78,7 +77,7 @@ def handle(self, *args, **options):
             print('Nothing to do!')
             return
 
-        num_threads = multiprocessing.cpu_count() * 2
+        num_threads = 2
```

Comment:
maybe this should be a parameter you can define in runtime. Default to 2.

Reply:
Good idea! I've parameterized it